graph/DataGraph.java

```diff
@@ -170,7 +170,7 @@ public ThingVertex create(TypeVertex typeVertex, boolean isInferred) {
         final ThingVertex vertex = new ThingVertexImpl.Buffered(this, iid, isInferred);
         thingsByIID.put(iid, vertex);
         thingsByTypeIID.computeIfAbsent(typeVertex.iid(), t -> new HashSet<>()).add(vertex);
-        statistics.vertexCreated(typeVertex.iid());
+        if (!isInferred) statistics.vertexCreated(typeVertex.iid());
```

Comment:
we should only additionally count non-inferred vertices? @lriuui0x0  what do you think of this